### ISSUE TITLE
[BUG] Latest release not working on newly upgraded server (part 2)

### ISSUE THREAD

**Issue by Eireman** [p4535992/foundryvtt-automated-polymorpher#24]
Update now works, however character does not change object. HOWEVER manually created actor of shape DOES work back to main character,

On initial transform from main character, action reports:

> commons.js:1696 Error: You are accessing BaseToken#vision which has been migrated to BaseToken#sight.enabled
> Deprecated since Version 10
> Backwards-compatible support will be removed in Version 12
>     at logCompatibilityWarning (commons.js:1685:19)
>     at BaseToken._logDataFieldMigration (commons.js:6601:14)
>     at Object.get (commons.js:6565:18)
>     at Object.prepareDataFromTransformOptions (dnd5e.js:933:45)
>     at Object.transformInto (dnd5e.js:109:30)
>     at async Object.callback (dnd5e.js:704:25)
> logCompatibilityWarning @ commons.js:1696
> _logDataFieldMigration @ commons.js:6601
> get @ commons.js:6565
> prepareDataFromTransformOptions @ dnd5e.js:933
> transformInto @ dnd5e.js:109
> await in transformInto (async)
> callback @ dnd5e.js:704
> await in callback (async)
> submit @ foundry.js:55745
> _onClickButton @ foundry.js:55674
> dispatch @ jquery.min.js:2
> y.handle @ jquery.min.js:2
> dnd5e.js:1075 Uncaught (in promise) TypeError: effect.toObject is not a function
> [Detected 1 package: automated-polymorpher]
>     at Object.prepareDataFromTransformOptions (dnd5e.js:1075:38)
>     at Object.transformInto (dnd5e.js:109:30)
>     at async Object.callback (dnd5e.js:704:25)

Dragging in the "wildshape" actor does let me use the reverse function, however.

For reference I have deleted and completely rebuilt the wildshape/polymorph list for this character with no success.

**Comment by p4535992** [comment]
Please specify always System and Foundryvtt version.
That is a warning not a error it can be ignored for now.
Seem to work on my test actors.
Please export (with the export json function) the two actors  json data (rename the file with ".txt" for github limitation on the attachments) with the problem and i will check out if the issue persist on my side.

**Comment by p4535992** [comment]
Try out the version 1.1.6 , made some bug fixes not sure if i solve your problem either.

**Comment by p4535992** [comment]
probably fixed with 1.1.8

**Comment by Eireman** [comment]
I am sorry for the late response. My oldest son passed last month and this weekend involved quite a bit of work on his hospital bills (cancer, so, A LOT), and ensuring his daughter is being well cared for.

We were using Foundry VTT v10.286 with plugin version 1.1.6RC and the issue continued. I am NOT including character .json as the issue was fixed with 1.1.8, which I just momentarily upgraded to and immediately tested. I did have to reset the data on the polymorph actors, but from there every single one works both to and from!

Thank you for your work, @p4535992 . As a former software developer (network engineer) I know the issues involved in trying to solve a bug and not getting responses from the person who ID'd the bug. Life is a little weird here for me and I didn't anticipate 2 hours of work turning into 250 miles of driving and the entirety of Friday through Sunday.
